packages/block-editor/src/components/inserter/block-types-tab.native.js

```diff
@@ -12,8 +12,14 @@ import { store as blockEditorStore } from '../../store';
 import useBlockTypeImpressions from './hooks/use-block-type-impressions';
 
 const NON_BLOCK_CATEGORIES = [ 'reusable' ];
-
-const ALLOWED_EMBED_VARIATIONS = [ 'core/embed' ];
+const ALLOWED_EMBED_VARIATIONS = [
+	'core/embed',
+	'core/embed/youtube',
+	'core/embed/twitter',
+	'core/embed/wordpress',
+	'core/embed/instagram',
```

Comment:
The Instagram block won't be available in the block picker until we include [its variation from Jetpack](https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/extensions/extended-blocks/core-embed/instagram.js).

Reply:
Thanks for noting this.